dog-age.js

```diff
@@ -0,0 +1,5 @@
+let dogYearOfBirth = 2018;
+let dogYearFuture =  2025;
+let dogYear = dogYearFuture - dogYearOfBirth;
+ shouldShowResultInDogYears = "true";
```

Comment:
This variable is missing the declaration part (var, let or const). Also boolean values are better written not as strings but simply as they are `true` instead of `"true"`